packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -313,6 +331,32 @@ const MessageThreadStory = (args): JSX.Element => {
     });
   };
 
+  const onClickHandler = (): void => {
+    console.info(selectedMessageType);
```

Comment:
Do we need to keep the console.info here?

Reply:
Removed!